pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "kaiba"
-version = "2.0.0"
+version = "2.0.1"
```

Comment:
This is a breaking change, not a minor version. It will break their production runs.

Version `3.0.0` will be more appropriate. @thomasborgen What do you think?

Reply:
@ChameleonTartu  Nothing about the interface or output from kaiba changes from these changes. I think `patch` is alright. Unless I missed something 🙈

Reply:
@thomasborgen Hm, 

Example,

Before `kaiba.casting.CastToDecimal` 
Now `kaiba.casting._cast_to_date.cast_to_date`

It breaks the contract as previously, `CastToDecimal` was not protected or private. In the meantime, if you use it only via `get_casting_function` this is a minor change.

I guess the decision is up to you, but from my perspective version upgrade should be `major`.

Reply:
Good point. 
I think it makes sense for us to define what the interface to kaiba is. The way I've been thinking about it is that its only the functions in `process.py`.

We can do major now, and make sure to document that the only interface that will be considered when deciding on change level is the functions in process.py. We could also move them to the main __init__.py and move everything else into private scope.